src/utils/utils.js

```diff
@@ -26,4 +26,23 @@ export function sortArray(sortType, arrayToSort) {
             break;
     };
     return sortedArray;
-};
\ No newline at end of file
+};
+
+export function getRndInteger(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) ) + min;
+}
+
+export function randomKeys(numberOfProducts) {
```

Comment:
Lets better use [faker lib and its method](https://fakerjs.dev/api/helpers.html#arrayelements).

Reply:
Updated